pkg/cmd/ui/ui.go

```diff
@@ -202,7 +202,7 @@ func EnterServiceWebhookSecret() string {
 }
 
 // SelectOptionImageRepository , allows users an option between the Internal image registry and the external image registry through the UI prompt.
-func SelectOptionImageRepository() string {
+func SelectOptionImageRepository() bool {
```

Comment:
`SelectOptionImageRepository` feels odd here now that it returns a bool...

How about `UseInternalRegistry` ?